src/Pages/SuweeMain/NewBook/NewBook.js

```diff
@@ -0,0 +1,99 @@
+import { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import { withRouter, Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { BOOKLIST } from '../../../config';
+import { BESTSELLER_SUBCATEGORY_DATA } from '../../SuweeMain/mainData';
+
+const NewBook = () => {
+  const [newBooks, setNewBook] = useState([]);
+  const [categoryButton, setCategoryButton] = useState(1);
+
+  let history = useHistory();
+
+  const handleClick = (selected) => {
+    history.push(`/detail/${selected.id}`);
+  };
```

Comment:
```suggestion
  const handleClick = ({ id }) => history.push(`/detail/${id}`);
```
함수 파라미터 자리에서 직접 구조분해 할당 가능합니다~

Reply:
🤭! 적용해보겠습니다!